tests/integration/actions/doc/base.py

```diff
@@ -49,33 +49,28 @@ def test(
         """test interactive config"""
         received_output = tmux_config_session.interaction(user_input)
 
-        # mask out some config that is subject to change each run
-        if expected_in_output:
-            received_output = "\n".join(received_output)
-
-            print(f"received_output is:\n{received_output}")
-            print(f"expected_in_output is:\n{expected_in_output}")
-            for out in expected_in_output:
-                assert out in received_output
-        else:
-            updated_received_output = []
-            for line in received_output:
-                mask = "X" * 50
-                if "filename" in line or "│warnings:" in line:
-                    updated_received_output.append(mask)
+        updated_received_output = []
+        for line in received_output:
+            mask = "X" * 50
```

Comment:
This can move outside of for loop

Reply:
right, I'll label gate and log an issue, TY